src/Sylius/Bundle/AddressingBundle/Controller/ZoneMemberController.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\AddressingBundle\Controller;
+
+use Sylius\Bundle\ResourceBundle\Controller\ResourceController;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+
+/**
+ * @author Jan Góralski <jan.goralski@lakion.com>
+ */
+class ZoneMemberController extends ResourceController
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function createNew()
+    {
+        $request = $this->getConfiguration()->getRequest();
```

Comment:
This is not how it works in ResourceController right now. This is not an action method. :)